templates/mlops/kubeflow/what-is-kubeflow.html

```diff
@@ -0,0 +1,571 @@
+{% endblock meta_description %}
+
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/1m-96RlGkbzFa1KaGAxYFpY0ztIlsTuViCjXSAlErQlg
+{% endblock meta_copydoc %}
+
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
+
+{% block content %}
+
+  <div class="p-strip is-shallow u-no-padding--bottom">
+    <div class="u-fixed-width">
+      <div id="notification" class="p-notification">
+        <div class="p-notification__content">
+          <h5 class="p-notification__title">Apply to join our preview for Managed Kubeflow on Azure!</h5>
+          <p class="p-notification__message">
+            <a href="https://ubuntu.com/engage/managed-kubeflow-preview?utm_source=marketo&utm_medium=website&utm_campaign=701N100000VDqOkIAL"
+               target="_blank"
+               rel="noopener noreferrer">Learn more and sign up.</a>
+          </p>
+        </div>
+      </div>
+    </div>
+  </div>
+  <section class="p-section--hero">
+    <div class="grid-row">
+      <div class="grid-col-2 grid-col-medium-1 u-no-padding--bottom p-section">
+        {{ image(url="https://assets.ubuntu.com/v1/6d72759e-kubeflow-logo.png",
+                alt="Kubeflow",
+                width="852",
+                height="175",
+                hi_def=True,
+                loading="auto") | safe
+        }}
+      </div>
+      <div class="grid-col-6 grid-col-medium-3">
+        <div class="grid-row">
+          <div class="grid-col-4 p-section--shallow">
+            <h1 class="u-no-margin--bottom">What is Kubeflow?</h1>
+            <p class="p-heading--2">Kubeflow is the open source machine learning MLOps platform</p>
+          </div>
+          <div class="grid-col-2">
+            <p>
+              Kubeflow enables you to develop and deploy machine learning models at any scale. It is a cloud-native application that runs on any cloud.
+            </p>
+            <p>
+              Charmed Kubeflow is Canonical's distribution. It is secure and seamlessly integrated with other leading tools such as MLflow.
+            </p>
+          </div>
+        </div>
+        <hr class="p-rule--muted" />
+        <a class="p-button--positive"
+           href="https://charmed-kubeflow.io/docs/install">Try out Charmed Kubeflow</a>
+        <a class="p-button" href="https://ubuntu.com/engage/mlops-toolkit">Read our MLOps toolkit</a>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="u-fixed-width">
+      <hr class="p-rule" />
+      <h2 class="p-section--shallow">Contributors to Kubeflow</h2>
+    </div>
+    <div class="row--25-75">
+      <div class="col">
+        <div class="p-logo-section">
+          <div class="p-logo-section__items">
+            <div class="p-logo-section__item">
+              {{ image(url="https://assets.ubuntu.com/v1/7577d797-google-logo.svg",
+                            alt="Goofgle",
```

Comment:
```suggestion
                            alt="Google",
```
I personally prefer 'Goofgle' :joy: